content/blog/zhihong/2022-06-24_[gsoc_2022]_arm_port_and_device_tree_support_phase_1.md

```diff
@@ -0,0 +1,313 @@
++++
+type = "blog"
+title = "[GSoC 2022] ARM port and device tree support Phase 1"
+author = "Zhihong Niu"
+date = "2022-06-24"
+tags = ["haiku", "software", "gsoc2022", "arm", "device driver"]
+
++++
+
+
+In the first stage of work, the following content was completed:
+
+1. Compile and run arm haiku on osx
+2. Fixed some runtime errors
+
+The following will show how to compile haiku on osx and run it on qemu (my version: hrev56168)
+
+
+# Build and run Haiku on OSX (Intel)
+
+## Preparation
+
+在开始前，由于 osx 默认的 apfs 文件系统不区分大小小写，所以首先得新建一块大小写敏感的 volume。
```

Comment:
I think we can keep the article in english only?

Reply:
fixed